code/modules/mob/living/carbon/human/death.dm

```diff
@@ -16,6 +16,7 @@
 /mob/living/carbon/human/death(gibbed)
 	if(stat == DEAD)
 		return
+

```

Comment:
Left over from when Hearty Punch prevented death. I can remove it if it's that bad.